T07/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text;
+using static System.Console;
+
+OutputEncoding = Encoding.Unicode;
+WriteLine ("┌───┬───┬───┬───┬───┬───┬───┬───┐");
+var blackCoins = " ♜ │ ♞ │ ♝ │ ♛ │ ♚ │ ♝ │ ♞ │ ♜ │";
+var blackPawns = " ♟ │ ♟ │ ♟ │ ♟ │ ♟ │ ♟ │ ♟ │ ♟ │";
+var whiteCoins = " ♖ │ ♘ │ ♗ │ ♕ │ ♔ │ ♗ │ ♘ │ ♖ │";
+var whitePawns = " ♙ │ ♙ │ ♙ │ ♙ │ ♙ │ ♙ │ ♙ │ ♙ │";
+var connector = "├───┼───┼───┼───┼───┼───┼───┼───┤";
+var verticalBar = "│";
+for (int i = 0, size = 8, padLength = 4; i < size; i++) {
+   Write (verticalBar);
+   Action todo = i switch {

```

Comment:
```suggestion
   var s = i switch {
      0 => blackCoins,
      1 => blackPawns,
      6 => whitePawns,
      7 => whiteCoins,
      < 6 => String.Concat (Enumerable.Repeat (verticalBar.PadLeft (4), 8)),
      _ => string.Empty,
   };
   Write (s);

```
This will be more cleaner